src/basic/errno-util.h

```diff
@@ -153,3 +153,10 @@ static inline bool ERRNO_IS_DEVICE_ABSENT(int r) {
                       ENXIO,
                       ENOENT);
 }
+
+/* Quite often we want to handle cases where the backing FS doesn't support extended attributes at all and
+ * where it simply doesn't have the requested xattr the same */
```

Comment:
the same what?

Reply:
… we want to handle cases … the same

Reply:
Yeah, doesn't sound like proper english to me. Imho, needs a "way" at the end.